src/scripts/orb.sh

```diff
@@ -5,18 +5,23 @@ if [ "$(id -u)" = 0 ]; then export SUDO=""; else # Check if we are root
   export SUDO="sudo";
 fi
 
+# Fix Cert error - https://www.omgubuntu.co.uk/2017/08/fix-google-gpg-key-linux-repository-error
```

Comment:
Added this because was getting some certs error without this - https://app.circleci.com/pipelines/github/shipyardbuild/shipyard/11908/workflows/12a56e3d-12e7-4486-95ea-2cdc20378d26/jobs/41713